crates/common/src/comments/mod.rs

```diff
@@ -211,6 +211,8 @@ impl<'ast> CommentGatherer<'ast> {
                 }
             }
             TokenKind::BlockComment { is_doc, .. } => {
+                // TODO(rusowsky): handle this logic upstream in solar
+                let is_doc = is_doc || token_text.starts_with("/**");
```

Comment:
? is this for `/*** ...` (more than 2 stars) comments?

Reply:
Actually I think we're correct: https://github.com/argotorg/solidity/blob/965166317bbc2b02067eb87f222a2dce9d24e289/liblangutil/Scanner.cpp#L471-L514